src/pages/pricing/index.tsx

```diff
@@ -12,10 +13,17 @@ import emailIsValid from 'utils/email-is-valid'
 export const getServerSideProps: GetServerSideProps = async function ({res}) {
   res.setHeader('Cache-Control', 's-maxage=1, stale-while-revalidate')
```

Comment:
we should probably remove this since it will edge cache dynamic prices and we probably want a "no cache" situation